app/models/manageiq/providers/redhat/infra_manager/refresh/parse/parser.rb

```diff
@@ -211,6 +211,8 @@ def self.host_inv_to_hardware_hash(inv)
       result[:model] = hw_info[:product_name]
     end
 
+    result[:number_of_nics] = inv[:host_nics].count if inv[:host_nics]
```

Comment:
In this case the count is run on an array not on an ActiveRecord so it should be ok.